Ednpoints.txt

```diff
@@ -0,0 +1,147 @@
+There are two ways how we think registration can be done:
```

Comment:
@Am4rel  what way of storing each person's data would you recommend to us - in a separate collection or in role collections? And how to do it better - one role per account, or several roles per account?

Reply:
it depends on document structure of these types of users
if they will only differ by role (admin, user, chef, courier), then it's better to store them in one common `users` collection

if they should have different structure, than we need separate collections

I tempted to use one common collection, but we will see later, after database structure is thought of